pkg/components/enterprise.go

```diff
@@ -76,74 +76,74 @@ var (
 	}
 
 	ComponentEsCurator = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:34a8486c7e0d646b17c788adb4bafd41d567a85be5c75f4bfe1d650ebc85d475",
+		Version: "v2.8.0",
+		Digest:  "sha256:620672d054a15e3195bf6b8caf2a2811aaabf2f1c485648f2ecc1f752c533974",
 		Image:   "tigera/es-curator",
 	}
 
 	ComponentEsProxy = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:9da0f6f55431b9d6737bd915083cf51e09fe537b2072681e8b88a0cee5695259",
+		Version: "v2.8.0",
+		Digest:  "sha256:1c2838ddbca6290bce9631f489a3dfa98ea6321333fe745af2623512777c2edc",
 		Image:   "tigera/es-proxy",
 	}
 
 	ComponentFluentd = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:b3921b735d220e10808362925e04500a19b1edc15a2c0108e5104a8317f51c52",
+		Version: "v2.8.0",
+		Digest:  "sha256:18bc903059709436bfc00ae9977dcfcf00aa74e2f319f0dfe565dd2e5ebaa1c8",
 		Image:   "tigera/fluentd",
 	}
 
 	ComponentGuardian = component{
-		Version: "v2.7.0-0.dev-50-g2b3fa78",
-		Digest:  "sha256:22928d9b9711004c65699eb6daff547ce92cd6bd31feb680faff58d0a2a06d24",
+		Version: "v2.8.0",
+		Digest:  "sha256:7bb05ffce01fad8dc9078125ece06dc3180df732a853c775b3ac3602bcfe7a42",
 		Image:   "tigera/guardian",
 	}
 
 	ComponentIntrusionDetectionController = component{
-		Version: "v2.7.0-0.dev-38-gbf021d6",
-		Digest:  "sha256:11047f506a1db4d5295c1bd94afbdd2682c81cda9b9fccaafca4730f813bd4ee",
+		Version: "v2.8.0",
+		Digest:  "sha256:87289932cfc98a08776d98723225ca53fe244be86bd7d41c92c8fb756d9c5106",
```

Comment:
For me `make gen-versions` gave `sha256:ad85cab58dc5a7b66f7807a9450b7ad19f61abe894c5cda7d2706d0ab7fd972a`